tests/test.yml

```diff
@@ -169,14 +169,14 @@
     command: >
       oc -n test-non-admin get role osbs-custom-build
     register: cmd_role
-    failed_when: "'No resources found' not in cmd_role.stderr"
+    failed_when: "'No resources found' in cmd_role.stderr"
```

Comment:
Not sure I understand why this is being changed.

The task is to make sure custom builds roles are not created in the non-admin namespace. The implementation is to run oc to try getting roles.

If there are none there, isn't that a success?

Reply:
You're right. This seems to be reversed. The tests were failing prior to this change which led us to think a newer version of `oc` is reporting the same state differently.

@rcerven, we need to investigate this a bit more.

Reply:
ok found out that in new version of openshift it actually has different error message : 
    "stderr": "Error from server (NotFound): role \"osbs-custom-build\" not found", 
so I'll check for that as well